assets/js/googlesitekit/components-gm2/Checkbox.stories.js

```diff
@@ -74,6 +88,16 @@ Disabled.args = {
 	disabled: true,
 };
 
+export const DisabledChecked = Template.bind( {} );
+DisabledChecked.args = {
+	id: 'googlesitekit-checkbox-9',
+	name: 'googlesitekit__checkbox-9',
+	value: 'value-9',
+	label: 'Disabled Checkbox',
```

Comment:
Let's amend the label to distinguish it from the unchecked version of the disabled checkbox:

```suggestion
	label: 'Disabled Checked Checkbox',
```